### PR TITLE
[ADD] cron_run_manually: Tool for allowing to run manually a programmed task

### PR DESCRIPTION
**Pull request by pedrobaeza**

### CODE REVIEW

**Comment by hbrunn** [comment]
:+1:  (verified that travis errors are unrelated)

**Comment by lmignon** [comment]
:+1:

File: cron_run_manually/model/ir_cron.py [hunk 1]
```diff
@@ -0,0 +1,82 @@
...
+    _inherit = 'ir.cron'
+
+    def run_manually(self, cr, uid, ids, context=None):
+        """
+        Run a job from the cron form view.
+
+        Cut and paste of code snippets from addons/base/ir/ir_cron.py
+        """
+        logger = logging.getLogger('cron_run_manually')
+        cr.execute(
```
Comment (foutoucour) [inline cron_run_manually/model/ir_cron.py]: why do you have to do sql query here? Can't you get the model instead?

**Comment by StefanRijnhart** [comment]
Checked any changes compared to 7.0: just a single import was changed and Spanish translations were added. Thanks for the port! :+1:

**Comment by pedrobaeza** [comment]
I merge then. Thanks for the reviews.

**Pull request merged**